Aplib.Tests/Desire/GoalTests.cs

```diff
@@ -121,13 +121,14 @@ public void Goal_WhereEvaluationIsPerformed_DoesNotInfluenceBelieveSet()
     public void Goal_WhereHeuristicsChange_UsesUpdatedHeuristics()
     {
         // Arrange
+        IBeliefSet beliefSetMock = Mock.Of<IBeliefSet>();
```

Comment:
I would simply name it beliefSet, but that's a nit.

Reply:
Agree